### DISCUSSION TITLE
Python stubs

### DISCUSSION THREAD

**Discussion by brycegbrazen** [Autodesk/maya-usd#3781] [Q&A]
Hi all,

Is there a way to get Python stubs for this repo right now? I'm curious if there's some available for use in my IDE.

**Comment by seando-adsk** [comment]
Are you referring to https://mypy.readthedocs.io/en/stable/stubs.html?
Sorry this is not something that we have support for.

Sean

**Reply by brycegbrazen** [reply]
Yep that's referring to. Is that something I should create an issue for that you guys could support in the future?

**Reply by wallworm** [reply]
@brycegbrazen we will discuss it internally. Thanks for asking.
